### PR TITLE
tenant invite email mention

### PR DESCRIPTION
**Pull request by MarkWoulfeNeo4j**

### CODE REVIEW

**Comment by MarkWoulfeNeo4j** [comment]
[Document - "Welcome to Aura" email](https://trello.com/c/TTpFgm52/216-document-welcome-to-aura-email)

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-aura-257.surge.sh

File: modules/ROOT/pages/platform/user-management.adoc [hunk 1]
```diff
@@ -85,7 +80,7 @@ Select the **Delete member** trash can icon next to the team member's name, and
 
 === Accepting an invite
 
-When invited to a team, a **Tenant invitation** modal will appear the next time you reload the Console. 
+When invited to a team, you will receive an email with a link to accept the invite. This link will direct you to the Aura Console, where a **Tenant invitation** modal will appear.
```
Comment (nvitucci) [inline modules/ROOT/pages/platform/user-management.adoc:83]:
```suggestion
When invited to a team, you will receive an email with a link to accept the invite. 
This link will direct you to the Aura Console, where a **Tenant invitation** modal will appear.
```

**Review by nvitucci — APPROVED** [review]
Small nitpick, otherwise LGTM

**Pull request merged**